src/components/OptionsList.js

```diff
@@ -196,7 +200,7 @@ class OptionsList extends Component {
 
     render() {
         return (
-            <View style={[styles.flexGrow0]}>
+            <View style={[this.props.listContainerStyles]}>
```

Comment:
You're talking about this line right @marcaaron?

Reply:
@TomatoToaster  @marcaaron Damn, I am really sorry for the silly mistakes in this PR, guys. Looks like the festive season here has taken a toll on me! 😄

Reply:
Thank you so much for your patience and the time to review this as well! :)